bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -111,6 +118,7 @@ pub mod pallet {
 
 		/// Weights for pallet dispatchables
 		type WeightInfo: WeightInfo;
+		type AccountToLocation: for<'a> TryConvert<&'a Self::AccountId, Location>;
```

Comment:
```suggestion
		type AccountToLocation: Convert<Self::AccountId, Location>;
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/23/commits/72b4508d0b0e925dc51cfffdaa1cc678a1e1330b I couldn't find a `convert` impl in Substrate that does AccountId to Location conversion. I added my own one in `location.rs`.